docs/self-managed/components/orchestration-cluster/core-settings/configuration/configuration-mapping.md

```diff
@@ -0,0 +1,94 @@
+
+:::
+
+### New properties and backwards compatibility
+
+Backwards compatibility between new 8.8 properties and existing legacy properties is as follows:
+
+| Type                                                                                                                                                          | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <span className="badge badge--new">New</span>                                                                                                                 | <p>New Camunda 8.8 property without an existing equivalent legacy property.</p>                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| <span className="badge badge--1-to-1">1-to-1</span>                                                                                                           | <p>New Camunda 8.8 property with an existing equivalent legacy property or set of properties.</p><p>Backwards compatibility is supported as follows:</p><p><ul><li><p>If you have defined the new property, it is used.</p></li><li><p>If you have not defined the new property, the legacy property is used (if you have defined it).</p></li></ul></p>                                                                                                                                                                                                  |
```

Comment:
From: New Camunda 8.8 property with an existing equivalent legacy property or set of properties.
To: New unified Camunda 8.8 property with an existing equivalent legacy property or set of properties.